techniques/client-side-script/SCR40.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Using the CSS prefers-reduced-motion query in JavaScript to prevent motion</title>
+  <link rel="stylesheet" type="text/css" href="../../css/editors.css">
+</head>
+<body>
+
+<h1>Using the <abbr title="Cascading Style Sheets">CSS</abbr> <code>prefers-reduced-motion</code> query in JavaScript to prevent motion</h1>
+
+<section id="meta">
+  <h2>Metadata</h2>
+  <p id="id"></p>
```

Comment:
does a build step fill in this content or is it accidentally left empty?

Reply:
i believe so, yes (either fills it in, or outright removes it)